backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -142,16 +143,40 @@ def assign_first_reviewer(self) -> None:
     def assign_next(self) -> None:
         reviewers = self.assignees.filter(role=ListRole.reviewer).order_by("order")
 
-        # All reviews have reviewed the draft destruction list
+        # All reviewers have reviewed the draft destruction list
         if self.assignee == reviewers.last().user:
             self.get_author().assign()
-            self.set_status(ListStatus.internally_reviewed)
+            status = (
+                ListStatus.ready_to_delete
+                if self.has_short_review_process()
+                else ListStatus.internally_reviewed
+            )
+            self.set_status(status)
             return
 
         current_assignee = self.assignees.get(user=self.assignee)
         next_reviewer = current_assignee.next()
         next_reviewer.assign()
 
+    def has_short_review_process(self) -> bool:
+        zaken_urls = self.items.all().values_list("zaak", flat=True)
+
+        zaken = Zaak.objects.filter(url__in=zaken_urls)
+        zaaktypes_urls = set(zaken.values_list("zaaktype", flat=True))
+
+        config = ArchiveConfig.get_solo()
+
+        return all(
+            [zaaktype in config.zaaktypes_short_process for zaaktype in zaaktypes_urls]
+        )
+
+    def determine_next_step_post_review(self, review: "DestructionListReview") -> None:
```

Comment:
Is there a reason for the long name? Can't it simply be "determine_next" step that does it's thing regardless of whether is has or has not been reviewed?

Checking for the review state just feels like one of the things it could do and act on instead of rely upon. Wouldn't reducing the amount of process trigger (ideally to 1) make the process a bit more simple and possibly reliable?

Reply:
I moved the method to the review so that it is more clear that it is the next step after a review!